aquarius/graphql.py

```diff
@@ -0,0 +1,55 @@
+import time
+
+from gql import gql, Client
+from gql.transport.aiohttp import AIOHTTPTransport
+
+from aquarius.events.util import get_network_name
+
+
+def get_number_orders(token_address, last_sync_block):
+    client = get_client()
+
+    last_block = get_last_block(client)
+    while last_block <= last_sync_block:
+        last_block = get_last_block(client)
+        time.sleep(2)
+
+    did_query = gql(
+        '{ datatokens(where: {id: "' + token_address.lower() + '"}) { orderVolume } }'
```

Comment:
There will be a small change here in the new subgraph (still wip) so it will look something like `'{ tokens(where: {id: "' + token_address.lower() + '"}) { orderCount} }'` . It's minor but we should keep this in draft until we have a proper version of subgraph

Reply:
for the moment, we don't even start subgraph, so all counts will be -1 due to connection errors. Since v4 is a "rolling" branch, I think we can merge it now and leave the exact query as a todo.